joomla/joomla-cms
# [com_finder] maps view: translate branches (solves #10260 - part 2)

#### Summary of Changes

Solves https://github.com/joomla/joomla-cms/issues/10260 (branches part)

> Load another language than English in back-end.
> Go to Smartsearch Content Maps equivalent page
> The equivalent filter "- Select Branch -" displays the filter types in English

After PR (example in portuguese):

![image](https://cloud.githubusercontent.com/assets/9630530/15076583/59a8ba46-13a2-11e6-8a8f-bc80fcf6e576.png)
#### Testing Instructions
1. Apply patch
2. Change admin language to other language than en-GB
3. Check the "Branch" filter in Components -> Smart Search -> Content Maps
